.github/workflows/test-readme.sh

```diff
@@ -0,0 +1,22 @@
+set -e
```

Comment:
Might be useful for stuff like this, although I have never used it myself: https://docs.python.org/3/library/doctest.html

Reply:
It looks like doctest is only for testing Python code, not shell scripts 🤔 I moved the invocation of the script to the pytests, I like that better I think.